src/web_app_skeleton/config/env.cr

```diff
@@ -1,20 +1,29 @@
-module Lucky::Env
-  extend self
+# Lucky manages environments using `LuckyEnv`. By default, development,
+# production and test are supported.
+#
+# Example:
+# ```
+# LuckyEnv.environment  # => "development"
+# LuckyEnv.development? # => true
+# LuckyEnv.production?  # => false
+# LuckyEnv.test?        # => false
+# ```
+#
+# New environments can be added using the `LuckyEnv.add_env` macro.
+#
+# Example:
+# ```
+# LuckyEnv.add_env :staging
+# LuckyEnv.staging? # => false
+# ```
+#
```

Comment:
Overall, I really like this. The only thing I think that's missing is mentioning that the value comes from `ENV["LUCKY_ENV"]`..

Reply:
I've amended the original commit. See https://github.com/luckyframework/lucky_cli/pull/655/commits/6acd9028931e5f2df70d80d7519e329106b9fd64.